x-pack/legacy/plugins/reporting/export_types/csv_from_savedobject/server/create_job/create_job.ts

```diff
@@ -94,7 +94,6 @@ function createJobFn(server: KbnServer): CreateJobFn {
       });
 
     return {
-      basePath: req.getBasePath(),
```

Comment:
This is from basePath being removed from `JobDocPayload` which all the different JobDocPayload types extend from.